src/types/module.ts

```diff
@@ -51,12 +51,6 @@ declare module 'nuxt/schema' {
   }
 }
 
-declare module '@nuxt/schema' {
```

Comment:
Not sure about this change. zdo you think this should be deleted, not merged with the other change?

Reply:
It already was duplicated in the other block :)